app/controllers/TrustStatusController.scala

```diff
@@ -128,8 +134,14 @@ class TrustStatusController @Inject()(
       case SorryThereHasBeenAProblem =>
         logger.warn(s"[tryToPlayback][Session ID: ${Session.id(hc)}] $utr unable to retrieve trust due to status")
         Future.successful(Redirect(routes.TrustStatusController.sorryThereHasBeenAProblem()))
-      case _ =>
-        logger.warn(s"[tryToPlayback][Session ID: ${Session.id(hc)}] $utr unable to retrieve trust due to an error")
+      case TrustServiceUnavailable =>
+        logger.warn(s"[tryToPlayback][Session ID: ${Session.id(hc)}] $utr unable to retrieve trust due to the service being unavailable")
+        Future.successful(Redirect(routes.TrustStatusController.unavailable()))
+      case ClosedRequestResponse =>
+        logger.warn(s"[tryToPlayback][Session ID: ${Session.id(hc)}] $utr unable to retrieve trust due to the service closing the request")
+        Future.successful(Redirect(routes.TrustStatusController.unavailable()))
+      case response =>
+        logger.warn(s"[tryToPlayback][Session ID: ${Session.id(hc)}] $utr unable to retrieve trust due to an error ${response}")
         Future.successful(Redirect(routes.TrustStatusController.down()))
```

Comment:
I am now confused about the difference between the `.down()` case and the `.unavailable()` case? They're both part of the same thing?
What content will the user now see?